iib/workers/tasks/opm_operations.py

```diff
@@ -487,6 +489,133 @@ def _serve_cmd_at_port(
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
 
 
+def get_list_packages(
```

Comment:
Consider renaming this to `get_operator_package_list` or something similar. `get_list_packages` sounds confusing to me

Reply:
While doing the review, I was confused by this as well.

Reply:
Changed - please review